app/components/dialogs/CreateLocationDialog.js

```diff
@@ -89,17 +102,30 @@ class CreateLocationDialog extends React.Component<Props, State> {
         perspective: '',
         isDefault: false,
         isReadOnly: false,
-        watchForChanges: Pro ? true : false,
+        watchForChanges: !!Pro,
```

Comment:
Please leave the previous check for PRO. It is more readable...

Reply:
agree